pandas/tests/io/test_clipboard.py

```diff
@@ -220,6 +220,21 @@ def test_read_clipboard_infer_excel(self, request, mock_clipboard):
         # excel data is parsed correctly
         assert df.iloc[1][1] == "Harry Carney"
 
+        # a null on the first column works
```

Comment:
Please make a separate test and link the issue

Reply:
Hi @phofl, just updated it, let me know if I should change something else, thanks